coder_sniffer/Drupal/Test/Arrays/ArrayUnitTest.php

```diff
@@ -35,23 +45,47 @@ public function getErrorList()
      * The key of the array should represent the line number and the value
      * should represent the number of warnings that should occur on that line.
      *
+     * @param string $testFile The name of the file being tested.
+     *
      * @return array(int => int)
      */
-    public function getWarningList()
+    public function getWarningList($testFile=null)
+    {
+        switch ($testFile) {
+        case 'ArrayUnitTest.1.inc':
+            return [
+                17 => 1,
+                22 => 1,
+                23 => 1,
+                24 => 1,
+                37 => 1,
+                42 => 1,
+                44 => 1,
+                59 => 1,
+                76 => 1,
+            ];
+        case 'ArrayUnitTest.2.inc':
+            return [];
+        }
+
+    }//end getWarningList()
+
+
+    /**
+     * Returns a list of test files that should be checked.
+     *
+     * @param string $testFileBase The base path that the unit tests files will have.
+     *
+     * @return array The list of test files.
+     */
+    protected function getTestFiles($testFileBase)
```

Comment:
no need to override this method. The parent will find *1.inc and *2.inc just fine. See for example DocCommentStarUnitTest

Reply:
Thanks for the feedback. I've reverted the change to the existing array unit tests and removed unneeded overrides